services/database/server.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * This server will be the main interface with the postgres database.
+ * This server will automatically handle all of the mapping tables.
+ */
+
+var Router = require("express-promise-router");
+var express = require("express");
+var DAO = require('./DAO.js');
+var schema = require('./schema.js');
+var bodyParser = require('body-parser')
+
+const app = express();
+const router = Router();
+app.use(router);
+app.use(bodyParser.urlencoded({ extended: false }))  
+app.listen(3000, () => {
+	console.log("Started server")
```

Comment:
Add PORT environment variable for easier updates (primarily for testing multiple components on the same machine).
nit - console.log(`Started server at Port : ${PORT}`)